code/modules/surgery/organs/organ_external.dm

```diff
@@ -847,28 +852,27 @@ Note that amputating the affected organ does in fact remove the infection from t
 	status &= ~ORGAN_BROKEN
 	return 1
 
-// I put these two next to each other to highlight that both exist. This should likely be resolved.
-/obj/item/organ/external/robotize()
+/obj/item/organ/external/robotize(company, make_tough = 0, convert_all = 1)
```

Comment:
Yeah, I already figured that's the potential problem last night; I just need to ensure it is on the player's side.

I'm unable to replicate the problem, locally, with the heads, so I'm unable to test my theory and people haven't been forthcoming on live-testing, but it appears some are finally starting to give info, so...